### ISSUE TITLE
Add two new buttons to 1st 'Add' window 

### ISSUE THREAD

**Issue by AndyatFocallocal** [focallocal/fl-maps#100]
Button 1:

On the Gather Map users will be able to click 'Other Positive Action' to submit activities not on our automatically approved list.

On the Resources Map users will be able to 'Offer a Resource' to help people homeless (i.e. CV help, or buying them a coffee and a chat). 

Both will need to be approved by an admin/mod before going live - to avoid drug sales, bar crawls, etc spamming our map. Likely implementation will be that pushing the button selects a hidden category creating a listing which needs to be approved by a mod/admin. The user then continues on to the next window to add details.

Button 2:
Next to this will be a button with a link editable by admin in the backend
Gatherings Map. [Activiites List](news.focallocal.org/activities)
Brighter Tomorrows Map. [Suggested Offerings](news.focallocal/helping-homeless-suggestions

**Comment by ydaniel98** [comment]
@AndyatFocallocal Could you rephrase what button 2 is suppose to be?

**Comment by AndyatFocallocal** [comment]
sorry for the delay, i missed notifications for some reason. 

this will be a link to an external list of suggestions. 

on the gatherings map it'll link to this page: http://news.focallocal.org/activities-list/ (the page will be moving in the near future). 

on the Brighter Tomorrows Map it'll link to another external page with a list of suggestions they can offer to support local people who are homeless

Both can be titled 'Suggestions', as the button will be positioned right next to button 1, it should be obvious what the suggestions are in relation to.

**Comment by ydaniel98** [comment]
@AndyatFocallocal The text is hardcoded for now. I forgot to enable it with i18n. I'll make a PR with those changes later.

**Comment by AndyatFocallocal** [comment]
sorry. i should have been clearer with the description. the 2 buttons should be in this window next to back and next

![image](https://cloud.githubusercontent.com/assets/16229696/23317349/46a5ee82-face-11e6-810f-b1c177d5c9d2.png)

**Comment by AndyatFocallocal** [comment]
the 'other positive action' would operate the same as any other category, except it would need an admin to approve it before going live. so it'd load the same chain of boxes to add details, and be assigned its own colour like any other category. same with the 'offer a resource' although i think that would also be automatically approved

this page wouldnt be necessary (except for the admin in the backend). sorry for not explaining clearly enough. let me know if its still not clear.

![Uploading image.png…]()

**Comment by AndyatFocallocal** [comment]
the wifi here is too slow to upload the image. it was of the 'add category window that you created' we can def use that function for admins if the backend doesnt already have that ability :)

**Comment by ydaniel98** [comment]
@AndyatFocallocal The same as the current setup, but with the new category button as part of the category list?

**Comment by AndyatFocallocal** [comment]
yeah, so it'd be part of the category list but the main way users would access it would be a button at the bottom. easiest solution would be to just add a category where activities/resources need to be approved, and also add a function to the button at the bottom which would select that category and move to the next window

so we'd have a row of 4 buttons at the bottom now :)

**Comment by ydaniel98** [comment]
@AndyatFocallocal Would you say this is done?

**Comment by AndyatFocallocal** [comment]
should have 4 buttons, and the text removed. the link there would be a 'suggestions' button

**Comment by AndyatFocallocal** [comment]
also noticed, the button doesn't appear to be working atm
